src/Logic/CommandLineConvert.cs

```diff
@@ -610,10 +614,13 @@ internal static bool BatchConvertSave(string toFormat, string offset, Encoding t
                                         s += "_" + className + format.Extension;
                                     outputFileName = FormatOutputFileNameForBatchConvert(s, sf.Extension, outputFolder, overwrite);
                                     File.WriteAllText(outputFileName, newSub.ToText(sf), targetEncoding);
+                                    exitLoop = true;
                                 }
                             }
                         }
                         Console.WriteLine(" done.");
+                        if (exitLoop)
+                            break;
```

Comment:
yeah, the reason I use a variable is that there is one that is located inside another for so... what you think?!